driver/host_info.cc

```diff
@@ -45,27 +47,30 @@ HOST_INFO::HOST_INFO(const char* host, int port)
     : HOST_INFO(host, port, UP, false) {}
 
 HOST_INFO::HOST_INFO(std::string host, int port, HOST_STATE state, bool is_writer)
-    : host{ host }, port{ port }, host_state{ state }, is_writer{ is_writer }
-{
-}
+    : host{host}, host_id{RDS_UTILS::get_rds_instance_id(host)}, port{port}, host_state{state}, is_writer{is_writer} {}
 
 // would need some checks for nulls
 HOST_INFO::HOST_INFO(const char* host, int port, HOST_STATE state, bool is_writer)
-    : host{ host }, port{ port }, host_state{ state }, is_writer{ is_writer }
-{
-}
+    : host{host}, host_id{RDS_UTILS::get_rds_instance_id(host)}, port{port}, host_state{state}, is_writer{is_writer} {}
 
 HOST_INFO::~HOST_INFO() {}
 
 /**
- * Returns the host.
+ * Returns the host endpoint.
```

Comment:
I think this comment line and the comment line on line 68 below can be removed because they appear to be redundant. Having the `@return` comment line appears to be good enough.